components/UpsellPlusModal.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <UModal
+    :title="$t('upsell_plus_modal_title')"
+    :ui="{
+      title: 'text-lg laptop:text-xl font-bold text-center',
+    }"
+    @update:open="onOpenUpdate"
+  >
+    <template #body>
+      <div class="w-full max-w-[420px] max-laptop:mx-auto">
+        <!-- Introduction -->
+        <div class="flex flex-col items-start mx-0 laptop:mx-6 gap-2 text-sm laptop:text-base">
```

Comment:
Can these 2 div can be merged into `#body` and place all the necessary class in `:ui="{ body: '...' }"`?